x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.test.tsx

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import * as React from 'react';
+import uuid from 'uuid';
+import { shallow } from 'enzyme';
+import { AlertInstances, AlertInstanceListItem, alertInstanceToListItem } from './alert_instances';
+import { Alert, AlertTaskState, RawAlertInstance } from '../../../../types';
+import { EuiBasicTable } from '@elastic/eui';
+
+const fakeNow = new Date('2020-02-09T23:15:41.941Z');
+const fake2MinutesAgo = new Date('2020-02-09T23:13:41.941Z');
+
+const mockAPIs = {
+  muteAlertInstance: jest.fn(),
+  unmuteAlertInstance: jest.fn(),
+  requestRefresh: jest.fn(),
+};
+
+beforeAll(() => {
+  jest.resetAllMocks();
+  global.Date.now = jest.fn(() => fakeNow.getTime());
```

Comment:
Yeah, all mocks are wiped after a suite is complete.
And I agree, I hate mocking time, but the only way to avoid it is to use DI to inject a function that calls `Date.now()` which always feels like overkill 🤷‍♂